apps/src/weblab/brambleHost.js

```diff
@@ -498,7 +498,7 @@ function load(Bramble) {
   bramble_ = Bramble;
 
   Bramble.load("#bramble", {
-    url: "//downloads.computinginthecore.org/bramble_0.1.26/index.html?disableExtensions=bramble-move-file",
+    url: "https://static.codeprojects.org/bramble_0.1.26/index.html?disableExtensions=bramble-move-file",
```

Comment:
We went with `static.codeprojects.org` instead of, say, `bramble.codeprojects.org` because it gives a fairly clear description of the characteristics and purpose of this subdomain, and we have a long-term desire to serve other student projects from this domain so there may be reason to host static content other than Bramble here.

I had to add `https` here because when running Web Lab in local development (http) we got an error in `postMessage` about a mismatched origin, where the protocol-relative origin here was http but our CloudFront distribution was automatically upgrading requests to https.  This was not a problem before because `downloads.computinginthecore.org` was allowing both http and https connections, but we've committed to avoiding this as a best practice in the future.  Tested this locally, it works great!